src/main/scala/mesosphere/marathon/core/task/tracker/InstanceStateOpProcessor.scala

```diff
@@ -14,4 +18,14 @@ import scala.concurrent.Future
 trait InstanceStateOpProcessor {
```

Comment:
Nothing against this diff but `InstanceStateOpProcessor` and later `val stateOpProcessor` are ugly names :sigh:

Reply:
They'll go in the next PR 🙂.